packages/@aws-cdk/aws-stepfunctions/lib/state-transition-metrics.ts

```diff
@@ -0,0 +1,58 @@
+import cloudwatch = require('@aws-cdk/aws-cloudwatch');
+
+/**
+ * Metrics on the rate limiting performed on state machine execution.
+ *
+ * These rate limits are shared across all state machines.
+ */
+export class StateTransitionMetric {
+    /**
+     * Return the given named metric for the service's state transition metrics
+     *
+     * @default average over 5 minutes
+     */
+    public static metric(metricName: string, props?: cloudwatch.MetricCustomization): cloudwatch.Metric {
+        return new cloudwatch.Metric({
+            namespace: 'AWS/States',
+            metricName,
+            dimensions: { ServiceMetric: 'StateTransition' },
+            ...props
+        });
+    }
+
+    /**
+     * Metric for the number of available state transitions.
+     *
+     * @default average over 5 minutes
+     */
+    public static provisionedBucketSize(props?: cloudwatch.MetricCustomization): cloudwatch.Metric {
```

Comment:
Shouldn't all these methods begin with `metric`?

Reply:
Did not do that because the containing object is explicitly only about metrics. But I suppose for consistency's sake it could be.